third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/trusted-clients-registry.js

```diff
@@ -143,163 +135,121 @@ TrustedClientsRegistryImpl.prototype.logger =
     GSC.Logging.getScopedLogger('PcscLiteServer.TrustedClientsRegistry');
 
 /** @override */
-TrustedClientsRegistryImpl.prototype.getByOrigin = function(origin) {
```

Comment:
Here and below is the main change: switching from `then` to `await` and from `xhrio` to `fetch`.